iceoryx_examples/icedelivery/CMakeLists.txt

```diff
@@ -72,6 +75,28 @@ set_target_properties(iox-ex-publisher-untyped-modern PROPERTIES
     POSITION_INDEPENDENT_CODE ON
 )
 
+add_executable(iox-ex-subscriber-typed-modern ./iox_subscriber_typed_modern.cpp)
+target_link_libraries(iox-ex-subscriber-typed-modern
+    iceoryx_posh::iceoryx_posh
+)
+set_target_properties(iox-ex-subscriber-typed-modern PROPERTIES
```

Comment:
You can add the properties to multiple targets at once.
```
set_target_properties(iox-ex-subscriber-untyped-modern iox-ex-subscriber-typed-modern PROPERTIES
    CXX_STANDARD_REQUIRED ON
    ...
)
```

You can even add "iox-ex-publisher-typed-modern" and "iox-ex-publisher-untyped-modern"  from above to save more.

Reply:
This was copying the structure used by the old examples. 
I can change it though if you prefer. I don't think it really matters though...

Reply:
According to a popular committer here i would say: I have no strong feelings about this ;-)